liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/InstallFeatureMojo.java

```diff
@@ -47,6 +47,13 @@
      */
     @Parameter
     private Features features;
+
+
+    /**
+     * Liberty output directory (<tT>WLP_OUTPUT_DIR</tt>).
+     */
+    @Parameter(property = "outputDirectory", defaultValue = "${project.build.directory}/liberty-alt-output-dir")
```

Comment:
Isn't outputDirectory already defined as a common parameter in BasicSupport with no default value? How does this parameter with the same name interact with that one?